backend/api/Controllers/RobotController.cs

```diff
@@ -257,6 +257,12 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] CreateRobotQuery rob
             _logger.LogError(e, "{message}", message);
             return StatusCode(StatusCodes.Status500InternalServerError, message);
         }
+        catch (RobotPositionNotFoundException e)
+        {
+            string message = "A suitable robot position could not be found";
+            _logger.LogError(e, "{message}", message);
+            return StatusCode(StatusCodes.Status500InternalServerError, message);
```

Comment:
"... for one or more of the desired tags"
Also this means that the entire mission will fail if one tag is missing, is this desired functionality?

Reply:
Ideally we'd like to know which points are supported when creating the mission. This will be possible with the echo mission planner once the database for the robot positions are in place. Then we can enforce that all Ready echo missions only contain valid tags. I suggest that we await that implementation and accept that the entire mission will fail in the meantime.

Reply:
Agreed